pkg/config/nodetreemodel/config.go

```diff
@@ -555,10 +557,22 @@ func (c *ntmConfig) AllSettingsBySource() map[model.Source]interface{} {
 }
 
 // AddConfigPath adds another config for the given path
-func (c *ntmConfig) AddConfigPath(_in string) {
+func (c *ntmConfig) AddConfigPath(in string) {
 	c.Lock()
 	defer c.Unlock()
-	c.logErrorNotImplemented("AddConfigPath")
+
+	if !filepath.IsAbs(in) {
+		in, err := filepath.Abs(in)
```

Comment:
This won't override the `in` parameter because it uses `:=` to redefine.
```suggestion
		var err error
		in, err = filepath.Abs(in)
```